app/controlplane/internal/data/ent/migrate/migrations/20240229202352.sql

```diff
@@ -2,7 +2,7 @@
 ALTER TABLE "memberships" ADD COLUMN "role" character varying;
 
 -- Set the existing memberships to the role "role:admin" to not to break compatibility
-UPDATE "memberships" SET "role" = 'role:admin';
+UPDATE "memberships" SET "role" = 'role:org:admin';
```

Comment:
note: this role was invalid, sorry I missed it. 

I updated it here since it has not been released yet.